projects/hslayers/src/components/layermanager/layermanager.service.ts

```diff
@@ -1175,4 +1176,25 @@ export class HsLayerManagerService {
       window.URL.revokeObjectURL(url);
     }, 0);
   }
+
+  /*
+    Creats a copy of the currentLayer
+  */
+  copyLayer(newTitle: string): void {
+    const copiedLayer = new VectorLayer(
+      this.currentLayer.layer.getProperties()
```

Comment:
Will need to also clone the layers Source. Currently the cloned layer and original layer shares the same source object, which might create problems - changing dimension or other properties in one changes other.